sdk-tests/src/test/java/io/dapr/it/methodinvoke/grpc/MethodInvokeIT.java

```diff
@@ -138,10 +136,12 @@ public void testInvokeException() throws Exception {
             DaprException exception = assertThrows(DaprException.class, () ->
                 client.invokeMethod(daprRun.getAppName(), "sleep", req.toByteArray(), HttpExtension.POST).block());
 
-            assertEquals("UNKNOWN", exception.getErrorCode());
             if (this.useGrpc) {
-                assertEquals("UNKNOWN: ", exception.getMessage());
+                assertEquals("INTERNAL", exception.getErrorCode());
+                assertNotNull(exception.getMessage());
+                assertTrue(exception.getMessage().contains("INTERNAL:"));
             } else {
+                assertEquals("UNKNOWN", exception.getErrorCode());
```

Comment:
Unrelated to this PR, but see @mukundansundar's comment from #789 

> related to https://github.com/dapr/dapr/pull/5648/files#diff-ce3f68305f0eb28b6b8911c281186536c80d8cac9dff3cece2c8bdf3dd955315R465-R475
now we get in gRPC (io.dapr.exceptions.DaprException: INTERNAL: fail to invoke, id: methodinvokeit-methodinvokeservice, err: message is nil)
instead of UNKNOWN